templates/rankings.html.j2

```diff
@@ -63,6 +63,38 @@
                     </tr>
                 {% endfor %}
             </table>
+
+            <footer>
+                <ul>
+                    <li id="human-like">
+                       Human-like: How similar<a href="#similarity">*</a> the average response from the model
+                       is to the <a href="/human.html">human written responses</a>.                    </li>
+                    <li id="cost">Cost: The average (mean) cost of one query.</li>
+                    <li id="consistency">
+                        Consistency:
+
+Consistency is computed in this way:
+
+<ol>
+    <li>
+    Get the embedding from each of a model's responses
+    </li><li>
+    Compute the similarity<a href="#similarity">*</a> between each of those and
+    the average of those embeddings.
```

Comment:
There's something about this sentence that still feels a bit off. Some kind of ambiguity about what's plural and what's singular?